src/main/resources/db/migration/changelog-v0.0.7.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Change this filename to the format `changelog-YYYYMMDDHHMM.xml`. So today at 10:15 AM would be `changelog-202303211015.xml`.